### ISSUE TITLE
Dose VALI support jetson devices？

### ISSUE THREAD

**Issue by zhjunqin** [RomanArzumanyan/VALI#43]
Dose VALI support jetson devices? Thanks.

**Comment by RomanArzumanyan** [comment]
Hi @zhjunqin 

Latest VALI doesn't support Jetson. 
There was a PR for VPF which did add Jetson support. It's not difficult from programmer's perspective. 

However, HW acceleration for video on Jetson requires a special FFMpeg version from Nvidia. It's available from a package manager but I didn't do any kind of research if that's compatible with VALI licensing. So the build procedure for Jetson may become complicated. Also I don't possess a Jetson board and would have to purchase it for development and in order to deploy a GitHub self-hosted runner for CI tests.

Long story short: it looks like a good feature request for VALI v.4. Please start a feature request conversation.
